quic/s2n-quic/src/tests/offload_tls.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#[test]
+#[cfg(feature = "unstable-offload-tls")]
+fn offload_tls() {
```

Comment:
looks pretty straightforward to use, but might still be worth an example in the `examples` directory. And that can contain some documentation on the feature in a readme